pkg/kubelet/kubelet.go

```diff
@@ -901,6 +907,74 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) getPodVolumes(podUID types.UID) ([]volumeTuple, error) {
+	podVolDir := kl.getPodVolumesDir(podUID)
+	volumeDirs, err := ioutil.ReadDir(podVolDir)
+	if err != nil {
+		glog.Errorf("Could not read directory %s: %v", podVolDir, err)
+	}
+	volumes := []volumeTuple{}
+	for _, volumeDir := range volumeDirs {
+		plugName := volumeDir.Name()
+		glog.Infof("get mounted volume plugName %q", plugName)
+		//plugName := strings.UnescapeQualifiedNameForDisk(volumeDir.Name())
+		volumePluginPath := path.Join(podVolDir, plugName)
+		// ioutil.ReadDir exits without returning any healthy dir when encountering the first lstat error
+		// but skipping dirs means no cleanup for healthy volumes. switching to a no-exit api solves this problem
+		volumeNameDirs, volumeNameDirsStat, err := util.ReadDirNoExit(volumePluginPath)
+		if err != nil {
+			return []volumeTuple{}, fmt.Errorf("could not read directory %s: %v", volumePluginPath, err)
+		}
+		for i, volumeNameDir := range volumeNameDirs {
+			glog.Infof("get mounted volume name %q", volumeNameDir.Name())
```

Comment:
nit: s/get/Get

Capitalizing the log messages is the convention in the codebase.

Again: add verbosity level